CHANGES_NEXT_RELEASE

```diff
@@ -17,3 +17,4 @@
 - Add: configuration service is exposed as part of the module
 - Add: casting attribute's values to JSON native types (activable by config.autocast)
 - Fix: GET call to northbound group endpoint returns multiple services (#604)
+- Fix: timestampProcessPlugin supports processing an array of NGSI entities (#621)
```

Comment:
Maybe it would be more precise?

> - Fix: timestampProcessPlugin supports processing an array of NGSI entities in NGSIv2 (#621)

(Of course, if I'm right -I'm not sure- and the problem only happens with NGSIv2. It could be considered part of the already existing "Add: allow NGSIv2 for updating active attributes..." but better to be explicit, so good idea to include a new line for this PR)

Reply:
I agree with you, it's better to have this entry in the CR as a separate line to improve traceability.

Reply:
Fixed in d0fafa981fb7f1363e8110b6483c0c0b8f7a7f33

Reply:
Typo: NGSIc2

Reply:
Fixed in e9b0fdd8a82a353db7d0e6c4d52617d20fad1c96